README.md

```diff
@@ -513,6 +513,34 @@ When perfection is achieved and there is nothing left to `delete`, you can do th
 delete delete!
 ```
 
+The `delete` function works globally. If you need to use a deleted function in some other project, you must `create` it again:
```

Comment:
Delete only delete the keyword not the action and only delete in the current program/file/function. You can also do that to get it back later
```js
const const _ = class!
delete class!!!
class Player {}! // Error: class was deleted
const const class = _! // Do nothing because less ! than the delete
class Player {}! // Error: class was deleted
const const class = _!!!
class Player {} // OK!
```